### ISSUE TITLE
Merging multiple models

### ISSUE THREAD

**Issue by jorisbalc** [nanoporetech/remora#153]
Greetings,

Is there any possibility to merge Remora models in a similar way that you have mC and hmC models together in a single one? With the newest Dorado update, it seems that multiple models of the same canonical base motif cannot be used alongside each other. The goal would be to call mC_hmC via a pre-trained Remora model in conjunction with a custom model for a third cytosine modification.

**Comment by marcus1487** [comment]
I think there might be a conceptual disconnect here. For the core target of distinguishing C, mC, hmC and another C modified base the Remora model should be trained with each of these possibilities presented to the model. This way the model can determine the signal characteristics of the each modified base compared to each other. 

Combining results after modeling mC/hmC and another C mod separately will always be more error prone in terms of accuracy across all categories. It is possible to run the two models separately and merge the results via some statistical procedure, but this is not recommended and thus no recommendations on how to best complete this merging is provided. 

If this 4 category model (C/mC/hmC/X) is required the best recommendation would be to produce training data for all 3 modified bases.

**Comment by jorisbalc** [comment]
Thank you for clearing that up. This way, would the optimal training data consist of separate reads, each with a unique modification or a mixture of all of them? I.e. 3 modified samples with 3 unique modifications or a single sample with a mixture of all of them?

**Comment by marcus1487** [comment]
Yes the proper way is as you described including examples of all modified bases at training time.

No, unfortunately the 5mC and 5hmC training data is not publicly available.

**Comment by jorisbalc** [comment]
Hey marcus. I've been thinking of ways to discern custom modifications and natural ones in vivo. Would it be reasonable to some extent to add methylated and hydroxymethylated reads along the canonincal bases during training? As far as I understand, training a model on three modifications would be quite challenging (specifically, the hmC part), perhaps this is a more simple way of ignoring the natural modifications, which later can be basecalled via ont's provided mC/hmC model if needed?
